### DISCUSSION TITLE
When downloading files with certain extensions, the AB Manager doesn’t get triggered/activated.

### DISCUSSION THREAD

**Discussion by Db2k8652** [amir1376/ab-download-manager#269] [Ideas]
During usage, I found that some file extensions don’t trigger the AB Manager. Could we consider adding settings for these extensions?

**Comment by amir1376** [comment]
Hi there.
You mean 
- browser does not send it to the app?
- or its category does not recognized in the app?

**Reply by Db2k8652** [reply]
Thank you for your reply. Yes, some file types like .e files and .ec files won’t trigger the AB Manager when downloading in the browser.

**Reply by rna0** [reply]
Hi, same happens to me with the mac .dmg extention

**Reply by petesimon** [reply]
did you add the those files types "e" and "ec" and "dmg" to the settings in the integration extension in your browser?

(left-click on the icon of integration extension in your browser and select "More Settings" ⚙️ )

**Reply by rna0** [reply]
Thank you, found it in more settings, for those who come after:
<img width="429" height="655" alt="image" src="https://github.com/user-attachments/assets/00c9e452-828e-4f3b-b166-0f36025b071a" />

**Comment by petesimon** [comment]
@Db2k8652 
please close this issue ticket
